spec/actions/bump_version_update_changelog_create_pr_action_spec.rb

```diff
@@ -167,7 +167,7 @@ def setup_stubs
       allow(Fastlane::Helper::RevenuecatInternalHelper).to receive(:validate_local_config_status_for_bump)
         .with('release/1.13.0', mock_github_pr_token)
       allow(Fastlane::Helper::VersioningHelper).to receive(:auto_generate_changelog)
-        .with(mock_repo_name, mock_github_token, 3)
+        .with(mock_repo_name, mock_github_token, 3, nil, nil)
```

Comment:
We should add a test making sure the parameters are passed correctly if they are passed.